pkgs/development/compilers/terra/default.nix

```
@@ -30,12 +30,12 @@ let
 
 in stdenv.mkDerivation rec {
   pname = "terra";
-  version = "1.0.0-beta3_${builtins.substring 0 7 src.rev}";
+  version = "1.0.0-beta5_${builtins.substring 0 7 src.rev}";
```

```suggestion
  version = "1.0.0-beta5";
```
We are phasing that pattern out of go packages.